src/loaders/markdown.mjs

```diff
@@ -16,16 +16,27 @@ const createLoader = () => {
    * Loads API Doc files and transforms it into VFiles
    *
    * @param {string} searchPath A glob/path for API docs to be loaded
+   * @param {string | undefined} ignorePath A glob/path of files to ignore
    * The input string can be a simple path (relative or absolute)
    * The input string can also be any allowed glob string
    *
    * @see https://code.visualstudio.com/docs/editor/glob-patterns
    */
-  const loadFiles = searchPath => {
-    const resolvedFiles = globSync(searchPath).filter(
+  const loadFiles = (searchPath, ignorePath) => {
+    let resolvedFiles = globSync(searchPath).filter(
       filePath => extname(filePath) === '.md'
     );
 
+    if (ignorePath) {
```

Comment:
I believe we can maybe clean this up so tha we don't need to keep reassigning the resolvedFiles variable with .filter()